docs/index.rst

```diff
@@ -20,9 +20,10 @@ and admin password were available, and, if and only if that file was changed,
 the appropriate service would be restarted:
 
 .. code-block:: python
+    from charms.core.templating import render
 
     @when('db.database.available', 'admin-pass')
-    def render_config(pgsql):
+    def render(pgsql):
         render_template('app-config.j2', '/etc/app.conf', {
```

Comment:
Thanks, @galgalesh! I'm closing this PR in favor of #94. Could you review that one as well?